geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -51,6 +51,10 @@ Information
 
 6. Now we needs the `pg_trgm` Postgres extension, use `CREATE EXTENSION pg_trgm;` to install it.
 
+7. With the new admin interface a mail server is required to send the user pasword (the administrator
```

Comment:
Suggestion:
7. A mail server is required to send the user his/her password (the application administrator no longer has access to this information).